README.md

```diff
@@ -54,7 +54,16 @@ jobs:
 
 ## FAQ - What are _Merge Changes_?
 
-During a merge, no matter the [strategy](https://git-scm.com/docs/merge-strategies), there may inadvertently be changes which can have negative side effects. For example, it may result in code that is no longer syntactically correct or checksums to be out of date.
+By default, as soon as the conflicts are resolved, the action will remove the assigned label. However, you can do it after requirements, like CI checks
+and reviews in protected branches are succesful. For this to work correctly, you must replace the events:
+
+```yml
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
AFAIK is no different from the example already provided

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

> Note: By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.

Which matches master https://github.com/prince-chrismc/label-merge-conflicts-action/blame/5750ede6000f5198154807b413f4ae6cafefa8dc/README.md#L28